Mu2eG4/src/Mu2eG4WorkerRunManager.cc

```diff
@@ -313,25 +318,32 @@ namespace mu2e {
     
     if(eventHasToBeSeeded)
     {
-      long seeds[3] = { s1, s2, 0 };
+      string msg = "r" + i_run + "s" + i_subrun + "e" + i_event + salt_;
+
+      string hash_out = CryptoPP_Hash(msg);
+      vector<string> randnumstrings = {hash_out.substr(0,8), hash_out.substr(8,8)};
```

Comment:
Making sure I understand...   The CryptoPP_Hash() function uses 8 bytes of the digest, but returns it as a hex number.  That is, 16 characters long.  So substr(8,8) does not go out of bounds.  Is that right?